LeezeIt/LeezeIt/PhoneNumberViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  PhoneNumberViewController.swift
+//  LeezeIt
+//
+//  Created by harris ali on 8/29/22.
+//
+import Foundation
+import UIKit
+class PhoneNumberViewController:UIViewController{
+    
+    @IBOutlet weak var phoneNumberText: UITextField!
+    @IBOutlet weak var leezeItI: UIImageView!
+    @IBOutlet weak var nextBut: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        nextBut.layer.cornerRadius = 30
```

Comment:
Instead of writing these modifiers in viewDidLoad() write them in the didSet of the Outlets so that viewDidLoad() stays clean

Reply:
resolved